app/views/subscriptions_management/update_address.html.erb

```diff
@@ -0,0 +1,58 @@
+<% content_for :title, 'Change your email address' %>
+
+<% content_for :head do %>
+  <meta name="robots" content="noindex, nofollow">
+<% end %>
+
+<% content_for :head_css do %>
+  <%= stylesheet_link_tag 'application-without-elements' %>
+<% end %>
+
+<%= render 'govuk_publishing_components/components/back_link', {
+  href: @back_url,
+} %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', title: 'Change your email address' %>
+  </div>
+</div>
+
+<% if flash[:error] %>
+  <div class="grid-row responsive-bottom-margin">
+    <div class="column-two-thirds">
+      <%= render 'govuk_publishing_components/components/error_summary', {
+        title: 'We weren’t able to change your email address',
+        items: [
+          {
+            text: 'There’s a problem with your email address',
+            href: '#email-address-input',
+          }
+        ]
+      } %>
+    </div>
+  </div>
+<% end %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/govspeak', {
+      content: %{<p>Your current email address is #{@address}</p>}
+    } %>
+    <%= form_tag change_address_path, method: :post do %>
+      <%= render 'govuk_publishing_components/components/input', {
+        error_message: flash[:error],
+        id: 'email-address-input',
+        label: { text: 'What’s your new email address?' },
+        name: :new_address,
+        type: 'email',
+        value: @new_address,
+      } %>
+      <%= render 'govuk_component/button', {
+        text: 'Save',
+        margin_bottom: true,
+      } %>
+      or <a href="<%= @back_url %>">cancel</a>
```

Comment:
The cancel link should be below the save button. This is for accessibility reasons: screen magnifier users are less likely to see the cancel option alongside the button. This positioning should also reduce the likelihood of clicking one in error.

Reply:
I've moved the link to be in the `info_text` of the button since it's related - let me know if this isn't the right thing to do

Reply:
How does that look?